resources/functionMap.php

```diff
@@ -6525,7 +6525,7 @@
 'ming_useconstants' => ['void', 'use'=>'int'],
 'ming_useswfversion' => ['void', 'version'=>'int'],
 'mkdir' => ['bool', 'pathname'=>'string', 'mode='=>'int', 'recursive='=>'bool', 'context='=>'resource'],
-'mktime' => ['int|false', 'hour='=>'int', 'min='=>'int', 'sec='=>'int', 'mon='=>'int', 'day='=>'int', 'year='=>'int'],
+'mktime' => ['__benevolent<int|false>', 'hour='=>'int', 'min='=>'int', 'sec='=>'int', 'mon='=>'int', 'day='=>'int', 'year='=>'int'],
```

Comment:
This function returns false only when it receives a value outside the integer range.

https://github.com/php/php-src/blob/df017cd0ef9e89d97b64dcf01efc5395fab30f6c/ext/date/php_date.c#L1209-L1213

PHP Manual says:

> `mktime()` returns the Unix timestamp of the arguments given, or false if the timestamp doesn't fit in a PHP integer.

It seems that even if `PHP_INT_MAX` is passed for all parameters, this value will never be exceeded.

https://3v4l.org/dnSu2